tests/foreman/cli/test_operatingsystem.py

```diff
@@ -358,31 +356,34 @@ def test_positive_update_parameters_attributes(self):
         assert param_name == os['parameters'][0]['name']
         assert param_value == os['parameters'][0]['value']
 
-    @pytest.mark.destructive
-    @pytest.mark.skip_if_open("BZ:1649011")
-    def test_positive_os_list_with_default_organization_set(self):
-        """list operating systems when the default organization is set
 
-        :id: 2c1ba416-a5d5-4031-b154-54794569a85b
+@pytest.mark.tier2
+@pytest.mark.skip_if_open("BZ:1649011")
+def test_positive_os_list_with_default_organization_set(satellite_latest):
+    """list operating systems when the default organization is set
 
-        :BZ: 1649011
+    :id: 2c1ba416-a5d5-4031-b154-54794569a85b
 
-        :expectedresults: os list should list operating systems when the
-            default organization is set
-        """
-        make_os()
-        os_list_before_default = OperatingSys.list()
-        assert len(os_list_before_default) > 0
-        try:
-            Defaults.add({'param-name': 'organization', 'param-value': DEFAULT_ORG})
-            result = ssh.command('hammer defaults list')
-            assert result.return_code == 0
-            assert DEFAULT_ORG in "".join(result.stdout)
-            os_list_after_default = OperatingSys.list()
-            assert len(os_list_after_default) > 0
-
-        finally:
-            Defaults.delete({'param-name': 'organization'})
-            result = ssh.command('hammer defaults list')
-            assert result.return_code == 0
-            assert DEFAULT_ORG not in "".join(result.stdout)
+    :BZ: 1649011
+
+    :expectedresults: os list should list operating systems when the
+        default organization is set
+    """
+    satellite_latest.api.OperatingSystem().create()
+    os_list_before_default = satellite_latest.cli.OperatingSys.list()
+    assert len(os_list_before_default) > 0
```

Comment:
> 0 is extraneous here, `len()` will always be positive, and 0 is falsy.

Reply:
yeah that's from the previous test code, but I can make it better. `assert os_list_before_default` should do fine.